lib/react_client.dart

```diff
@@ -192,11 +193,11 @@ final ReactDartInteropStatics _dartInteropStatics = (() {
   Component initComponent(
           ReactComponent jsThis,
           ReactDartComponentInternal internal,
-          InteropContextValue context,
+          dynamic context,
```

Comment:
This should be reverted since we only want to expose the new context stuff from `ReactDartInteropStatics2` class being created in #161.